docs/UserGuide.md

```diff
@@ -180,8 +180,9 @@ IDs for insurance plans:
 * `INSURANCE_ID` match the associated insurance plan. For a list of mappings between insurance plans and 
 IDs, refer above.
 * `CLAIM_ID` is generated by your insurance company and should be given to you before adding the claim. However, it does not 
-affect the functionality here except for deleting this claim later, where it would be required.
-* `CLAIM_AMOUNT_IN_DOLLARS_AND_CENTS` is the claim amount written in `DD.CC` format eg. `151.10`.
+affect the functionality here except for deleting this claim later, where it would be required. Note that the claim ID is always
+capitalised in the system so `b1234` is the considered the same as `B1234`. Do also note that to simulate real world conditions, we have fixed the claim ID to always follow a specific format which is `alphabet + 4 digit number`. All other format of claims will be considered invalid.
+* `CLAIM_AMOUNT_IN_DOLLARS_AND_CENTS` is the claim amount written in `DD.CC` format eg. `151.10`. So it can contain any number of digits for dollars but must be exactly 2 digits for cents, which is to ensure that the cents has been correctly indicated. Higher level of input validation is done here as money involved in claims is quite important information and this helps the user keep track of claim information more precisely and accurately.
```

Comment:
Should "Higher level of input validation is done here as money involved in claims is quite important information and this helps the user keep track of claim information more precisely and accurately." be placed in DG instead of UG?

Reply:
actually this thing should be in DG you are right

Reply:
Makes sense, since to the user (your average insurance agent) what the inner workings of the app are dosen't really matter 😅